packages/terra-table/CHANGELOG.md

```diff
@@ -3,8 +3,6 @@ Changelog
 
 Unreleased
 ----------
-### Changed
```

Comment:
Did this accidentally get removed after a merge?

Reply:
Yep! Worked with @bjankord on appropriate correction.